__test__/add_remove.test.js

```diff
@@ -35,4 +38,52 @@ describe('Add, Remove Element Test', () => {
     deleteElement(1);
     expect(JSON.parse(localStorage.getItem('List'))).toHaveLength(2);
   });
+
+  // Part 2
```

Comment:
[OPTIONAL] It would be better if you had separate `describe()` blocks for each functionality you are testing. This will make the testing codebase more organized and it'll give future reviewers that need to maintain the code you built a clearer idea of how the codebase is organized, which is the most important aspect of testing. I recommend you get in the habit of having several `describe()` blocks in your test suites.

Reply:
Thank you very much for your valuable suggestion. Sure, we will implement your recommendations.